### DISCUSSION TITLE
What is "collection songs" exactly?

### DISCUSSION THREAD

**Discussion by okaniso** [strawberrymusicplayer/strawberry#1559] [Q&A] [answered]
I have all my songs locally as ogg vorbis files. But when I try to add a cover for a song I often get
"Album cover editing is only available for collection songs."
So my question is, exactly what are the conditions for a song to be a collection song?
(I've tried to find a pattern when it works and when it doesn't, but I failed.)

I'm using strawberry version 1.0.14 (from Debian bookworm)

**Comment by jonaski** [comment, marked as answer]
Songs in directories that are added through Tools / Settings / Collection with info tracked in the database.
